primitives/consensus/beefy/src/witness.rs

```diff
@@ -80,15 +83,36 @@ mod tests {
 	use super::*;
 	use codec::Decode;
 
-	use crate::{crypto, known_payloads, Payload, KEY_TYPE};
+	use crate::{ecdsa_crypto, known_payloads, Payload, KEY_TYPE};
+
+	#[cfg(feature = "bls-experimental")]
+	use crate::bls_crypto::Signature as BLSSignature;
+
+	#[cfg(feature = "bls-experimental")]
+	use w3f_bls::{
+		single_pop_aggregator::SignatureAggregatorAssumingPoP, Message, SerializableToBytes,
+		Signed, TinyBLS377,
+	};
 
 	type TestCommitment = Commitment<u128>;
-	type TestSignedCommitment = SignedCommitment<u128, crypto::Signature>;
+
+	///types for ecdsa signed commitment
+	type TestSignedCommitment = SignedCommitment<u128, ecdsa_crypto::Signature>;
 	type TestSignedCommitmentWitness =
-		SignedCommitmentWitness<u128, Vec<Option<crypto::Signature>>>;
+		SignedCommitmentWitness<u128, Vec<Option<ecdsa_crypto::Signature>>>;
+
+	#[cfg(feature = "bls-experimental")]
+	#[derive(Clone, Debug, PartialEq, codec::Encode, codec::Decode)]
+	struct ECDSABLSSignaturePair(ecdsa_crypto::Signature, BLSSignature);
```

Comment:
Please use camel case for all novel type names (also for acronyms like ECDSA and BLS).
This is a de-facto standard practice in Substrate and Rust in general (see https://rust-lang.github.io/api-guidelines/naming.html)

```suggestion
	struct EcdsaBlsSignaturePair(ecdsa_crypto::Signature, BlsSignature);
```

Reply:
CamelCased!